corehq/apps/api/resources/v0_4.py

```diff
@@ -120,6 +120,7 @@ def obj_get_list(self, bundle, domain, **kwargs):
 
         def wrapper(doc):
             if doc['doc_type'] in xform_doc_types:
+                doc.pop('user_id', None)  # needed when xform is stored in sql
                 return xform_doc_types[doc['doc_type']].wrap(doc)
```

Comment:
fwiw, i'd prefer to avoid wrapping a sql xform instance as a couch instance as it could cause some potentially very confusing errors in the future. the couch instance adds some methods that should never be called on a sql form, such `.save`. if that were called then we'd end up with a sql xform and a duplicate couch xform. other methods would also be broken, but probably have less ramifications.

it's come up before that we needed to be able to wrap a sql xform, this might be the opportunity to try and make one of those kinds of functions. don't know enough about tastypie to know if there's a workaround